integration/forks/test_helpers

```diff
@@ -1,75 +1,72 @@
 #!/bin/bash
 
+source ../common
+
 build_images() {
   docker build -t geth ./geth
-	docker build -t smartcontract/chainlink ../../
+  docker build -t smartcontract/chainlink ../../
 }
 
 start_network() {
-	echo "Starting Network. See logs/{chain_[12],geth}.log for logs"
-	( docker-compose up > /dev/null ) &
+  echo "Starting Network. See logs/{chain_[12],geth}.log for logs"
+  ( docker-compose up > /dev/null ) &
 }
 
 tear_down() {
-	docker-compose down
-	# Remove CL database
-	sudo rm -rf tmp
+  docker-compose down
+  # Remove CL database
+  sudo rm -rf tmp
 }
 
 login() {
-  until docker exec -it forks_chainlink chainlink admin login -f /run/secrets/apicredentials
-  do
-      sleep 0.1
-  done
+  docker exec -it forks_chainlink chainlink admin login -f /run/secrets/apicredentials
 }
 
 create_job() {
+  login
   ETH_LOG_JOB=`cat fixtures/eth_log_job.json`
-	# wait for CL to start
-	search_chainlink_logs 'Listening and serving HTTP on port 6688'
-	# add ethlog job
-	login
-	docker exec -it forks_chainlink chainlink jobs create "$ETH_LOG_JOB"
+  docker exec -it forks_chainlink chainlink jobs create "$ETH_LOG_JOB"
 }
 
 create_contract() {
   CONTRACT_DATA=`cat fixtures/create_contract.json`
-	`curl \
-		-X POST \
-		-H "Content-Type: application/json" \
-		--data "$CONTRACT_DATA" \
-		172.16.1.100:8545`
+  curl \
+    -X POST \
+    -H "Content-Type: application/json" \
+    --data "$CONTRACT_DATA" \
+    172.16.1.100:8545
 }
 
 initial_setup() {
   tear_down
-	build_images
+  build_images
   start_network
+  waitForResponse "172.16.1.102:6688"
   create_job
 }
 
 
 assert_not_in_chainlink_logs() {
-	echo "asserting \"$1\" not present ..."
-	num_found=`docker-compose logs chainlink | grep -c "$1"`
-	if [ $num_found != "0" ]; then
-		printf "\n"
-		echo "TEST FAILED - Found $1 in chainlink logs and shouldn't have"
-		exit 1
-	fi;
+  echo "asserting \"$1\" not present ..."
+  num_found=`docker-compose logs chainlink | grep -c "$1"`
+  if [ $num_found != "0" ]; then
+    printf "\n"
+    echo "TEST FAILED - Found $1 in chainlink logs and shouldn't have"
+    exit 1
+  fi;
 }
 
 search_chainlink_logs() {
 	echo "searching for \"$1\" ... "
 	check_count=0;
 	TIMEOUT=30
 	until ( docker-compose logs chainlink | grep "$1" > /dev/null) ; do
-		if [ $check_count -gt $TIMEOUT ]; then
-			printf "\n"
-			echo "TEST FAILED - Timed out searching chainlink logs for \"$1\""
-			exit 1
-		fi;
-		check_count=$((check_count + 1))
-		sleep 1;
+    if [ $check_count -gt $TIMEOUT ]; then
```

Comment:
Are you using tabs here? The indentation looks wrong in github.

Reply:
Do we have standard bash linters / formatters we could use?

Reply:
Yeah i think my editor defaulted to tabs for some reason, but I changed it to all spaces.